crates/core/tedge_mapper/src/c8y/converter.rs

```diff
@@ -150,42 +154,30 @@ where
         input: &Message,
     ) -> Result<Vec<Message>, ConversionError> {
         let tedge_event = ThinEdgeEvent::try_from(input.topic.name.as_str(), input.payload_str()?)?;
+        let c8y_event = C8yCreateEvent::try_from(tedge_event)?;
 
         // If the message size is well within the Cumulocity MQTT size limit, use MQTT to send the mapped event as well
         if input.payload_bytes().len() < self.size_threshold.0 {
-            let smartrest_alarm = event::serialize_event(tedge_event)?;
-            let smartrest_topic = Topic::new_unchecked(SMARTREST_PUBLISH_TOPIC);
+            if c8y_event.extras.is_empty() {
+                let smartrest_event = format!(
+                    "{},{},\"{}\",{}",
+                    CREATE_EVENT_SMARTREST_CODE,
+                    c8y_event.event_type,
+                    c8y_event.text,
+                    c8y_event.time
+                );
```

Comment:
The `event::serialize_event` function has been deleted, but I think that an equivalent function would be useful to make the code clearer.